Makefile

```diff
@@ -16,7 +16,7 @@ ifdef BUILD_PLATFORM
 endif
 
 ifdef PHTHON_PATH
```

Comment:
Oh, there is a typo here, can you fix it in this pull request?
"PHTHON" -> "PYTHON"

Reply:
I've just fixed it.